File: src/model/Money.java
package model;

import model.common.WalletObject;

public class Money extends WalletObject {
    private double amount;

    public Money(int id, double amount) {
        super(id);
        this.amount = amount;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    @Override
    public String toString() {
        return "Money{" +
                "amount=" + amount +
                '}';
    }
}

File: src/model/Money.dart
import './common/wallet_object.dart';

class Money extends WalletObject {
  double amount;

  Money(int id, this.amount) : super(id);

  double getAmount() {
    return amount;
  }

  void setAmount(double amount) {
    this.amount = amount;
  }

  @override
  String toString() {
    return 'Money{amount=$amount}';
  }
}
